README.md

```diff
@@ -2,8 +2,166 @@
 
 Create and verify W3C Verifiable Credentials and Presentations in JWT format
 
-## API
+## Installation
 
-`createVerifiableCredential()`
+Using NPM:
 
-`createPresentation()`
\ No newline at end of file
+```
+npm install did-jwt-vc
+```
+
+Using Yarn:
+
+```
+yarn add did-jwt-vc
```

Comment:
we probably don't need to specify `yarn`

Reply:
removed